core/src/test/java/com/klaytn/caver/feature/RpcTest.java

```diff
@@ -212,13 +212,23 @@ public void testGetBlockByNumber() throws Exception {
                 true).send();
         KlayBlock.Block result = response.getResult();
         TestCase.assertEquals(testBlock.blockHash, result.getTransactions().get(0).getBlockHash());
+
+        response = caver.klay().getBlockByNumber(
+                new DefaultBlockParameterNumber(testBlock.blockNumber),
+                true).send();
```

Comment:
I think you need to create tests which has option(isFullTransaction)`false`. This is because issue #58 is related with the `false` option.

Reply:
It is an obvious mistake. I thought I made a false test. I'll fix it.